JessThrowaway/brand-new-api
# Development to main

Creating an API with Nestjs

To test:
Run application, then run postman with the GET call: http://localhost:3000/users/user-name, and you can replace 'user-name' with 'user-age' and 'user-color'. Should return values for the users.
